include/vcml/debugging/gdbserver.h

```diff
@@ -35,10 +35,19 @@ enum gdb_signal {
     GDBSIG_KILL = 9,
 };
 
+struct gdb_target {
+    u64 tid;
+    u64 pid;
+    string xml;
+    vector<const cpureg*> cpuregs;
+    target* tgt;
+};
+
 class gdbserver : public rspserver, private subscriber, private suspender
 {
 private:
-    target& m_target;
+    vector<gdb_target*> m_targets;
```

Comment:
Can this be `vector<gdb_target>`? Or why is this a pointer?

Reply:
done